jdi-light/src/main/java/com/epam/jdi/light/elements/complex/ListBase.java

```diff
@@ -45,7 +45,7 @@ public WebList list() {
     }
 
     ListBase() {}
-    ListBase(By locator) { list = new WebList(locator); }
+    ListBase(By locator) { list = new WebList(locator, 1); }
```

Comment:
new WebList(locator); (you can send 1 inside that method instead of make it through external public interface) , so user don't need to set index to use WebList with 1 by default